src/test/regress/pg_regress_multi.pl

```diff
@@ -56,6 +60,10 @@ ()
 my %functions = ();
 my %operators = ();
 my $valgrind = 0;
+my $valgrind_path = "valgrind";
```

Comment:
Should we use hyphen "-", or underscore "_" in variables? We should also update prints according to this above.